docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -705,47 +705,38 @@ Once clicked, a File Explorer will open, and you can select a name and location
 
 Adding Resource Files
 =============================
-Occasionally, it is necessary to manually add module resource files to the Control Center. These will be provided by a member of the Vertiq team. 
+Occasionally, it is necessary to manually add module resource files to the Control Center. These will be provided by a member of the Vertiq team as a .zip file. 
+Store this .zip file in a location that is easily accessible, such as Downloads, Desktop, or Documents.
 The process for integrating the files into the Control Center is described below for each supported operating system.
 
-Windows
-----------------
-1. Use the File Explorer to locate the Control Center's installation. By default this is Program Files (x86)/IQ Control Center
-2. Open the installed folder, and you'll see
+.. note::
+    The following instructions apply to all supported operating systems: Windows, Mac, and Linux (Ubunutu). The first step requires the Control Center application to be opened with administrator privileges.
+    On Mac and Linux, you must use the 'sudo' command in the terminal to run the application.
 
-.. image:: ../_static/control_center_pics/windows/windows_installed_folder.png
+1. Use the File Explorer to locate the Control Center's installation. By default, the application is located in Program Files (x86)/IQ Control Center/IQ Control Center/IQ Control Center.exe
+2. Right click the executable and select 'Run as administrator'.
 
-3. Enter the IQ Control Center folder, and you’ll see the Resources folder
+.. image:: ../_static/control_center_pics/windows/windows_run_as_administrator.png
 
-.. image:: ../_static/control_center_pics/windows/resources_windows.png
+.. note::
+    You need to run IQ Control Center as an administrator on Windows in order to import resource files.
 
-4. Inside of the Resources folder you’ll find Defaults, Firmware, and Tabs
-5. If you have been given a resource file with a number.json (42.json for example), copy the file into the Firmware folder
-6. If you have been given files with the names advanced, general, testing, or tuning, copy the files into the corresponding folder inside of the Tabs folder
+3. Once the application runs, navigate to the Menu in the top left and select 'Import Resource Pack'.
 
-Linux (Ubuntu)
-------------------
-1. Open a new terminal window, and navigate to the IQ Control Center installation. By default this is Home/~user~/IQ Control Center
-2. Inside you will see the following
+.. image:: ../_static/control_center_pics/windows/windows_menu_import_resource_pack.png
 
-.. image:: ../_static/control_center_pics/linux/linux_install_folder.png
+4. A popup will appear requesting that you run Control Center as an administrator. Since you already did this in step 1, click 'OK' and proceed to the next step.
```

Comment:
As a future control center idea...is there an easy way for us to tell if the app has already been open as an admin so we can avoid this popup if it's unnecessary?

Reply:
Currently it does not show any indication that the app is running with admin privileges but this is a good idea. One way would be to try running a command that requires admin privileges and display something on the application depending on if it is successful or not.